src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -660,7 +659,7 @@ TxQ::apply(Application& app, OpenView& view,
     // or transaction replacement, so just pull it up now.
     // TODO: Do we want to avoid doing it again during
     //   preclaim?
-    auto const baseFee = calculateBaseFee(app, view, *tx, j);
+    auto const baseFee = calculateBaseFee(view, *tx);
```

Comment:
`baseFee` is used in a single place. I think it could be inlined without any loss of readability (given its name)

Reply:
It's used twice. Line 664 and line 982. This function is too long, but refactoring feels out of scope for this PR. I could inline it twice, but that seems wasteful.

Reply:
That’s fine then. I missed it when going through the code (which says something about this function or about my eyesight).